api/libretime_api/legacy/migrations/0023_3_0_0_alpha_9_1.py

```diff
@@ -5,11 +5,11 @@
 from ._migrations import legacy_migration_factory
 
 UP = """
-ALTER TABLE cc_files ADD COLUMN artwork VARCHAR(4096);
```

Comment:
This is now a blank migration. Can it be removed completely?

Reply:
Then add a comment explaining why it needs to stay to avoid it being removed in future

Reply:
Added a comment in latest push